ecommerce/extensions/basket/views.py

```diff
@@ -379,14 +381,16 @@ def get_context_data(self, **kwargs):
         except ValueError:
             total_benefit = None
 
+
```

Comment:
Blank line is not need could be removed.

Reply:
done